tests/performance/framework.go

```diff
@@ -72,3 +72,7 @@ func skipIfNoRealtimePerformanceTests() {
 		Skip("Realtime performance tests are not enabled")
 	}
 }
+
+func KWOKDescribe(text string, args ...interface{}) bool {
+	return Describe(text, Label("KWOK", "sig-performance"), Serial, args)
```

Comment:
pull-kubevirt-check-unassigned-tests complains about the missing `[sig-performance]`
```suggestion
	return Describe("[sig-performance]"+text, Label("KWOK", "sig-performance"), Serial, args)
```

Reply:
Done, PTAL